apps/studio/src/server/modules/asset/asset.router.ts

```diff
@@ -39,8 +46,13 @@ export const assetRouter = router({
 
   deleteAssets: protectedProcedure
     .input(deleteAssetsSchema)
-    .mutation(async ({ ctx, input }) => {
-      const { fileKeys } = input
+    .mutation(async ({ ctx, input: { siteId, resourceId, fileKeys } }) => {
+      await validateUserPermissionsForAsset({
+        siteId,
+        resourceId,
+        action: "read",
```

Comment:
[issue] this should probably be an `update`?

Reply:
updated to be `delete`!